tests/connector_ssh/conf_engine/contactgroups.cfg

```diff
@@ -0,0 +1,16 @@
+###################################################################
+#                                                                 #
+#                       GENERATED BY CENTREON                     #
+#                                                                 #
+#               Developed by :                                    #
+#                   - Julien Mathis                               #
+#                   - Romain Le Merlus                            #
+#                                                                 #
+#                           www.centreon.com                      #
+#                For information : contact@centreon.com           #
+###################################################################
+#                                                                 #
+#         Last modification 2022-03-31 16:04                      #
+#         By admin_admin                                          #
+#                                                                 #
```

Comment:
if you don't use this file, you just have to remove it and remove its call in centengine.cfg